src/components/combos/PostCombo.js

```diff
@@ -144,7 +144,21 @@ const PostCombo = ({ userCombo, setUserCombo }) => {
               type="number"
               value={yearEstablished}
               placeholder={new Date().getFullYear()}
-              onChange={(e) => setYearEstablished(e.target.value)}
+              onChange={(e) => {
+                var value = parseInt(e.target.value)
+                if (value >= 0 && value <= new Date().getFullYear()) {
+                  setYearEstablished(value)
+                } else {
+                  setYearEstablished(new Date().getFullYear())
```

Comment:
Thank you for putting some thought into this! :) 
My intention behind resetting the year to current is that someone inserting a negative number probably made a mistake and it would be better to reset to the current year than to 2000 years ago ;)  
Imagine not realizing your mistake and then the trick is stored with the year 0. Does that make sense to you?

Reply:
Fair point. The argument could be made, that having a trick from the year that Jesus was born in, would then make it more clear that the user made a mistake, than it being from a pretty plausible but not necessarily correct year. 

Both options seem to have their pros and cons, you are making a very good point though. So as I said, feel free to ignore the comment, I just wanted to make sure that it wasn't an oversight :)

Reply:
Okay, thanks for clarifying. Personally, I would nevertheless stick to the "reset-to-current-year" solution. But changing this in the future would be no problem :)